src/main/java/com/minecolonies/coremod/network/messages/server/BuildToolPlaceMessage.java

```diff
@@ -205,8 +205,7 @@ private static void handleHut(
             return;
         }
 
-        final String hut = sn.getSection();
-        final ItemStack stack = BuildingUtils.getItemStackForHutFromInventory(player.inventory, hut);
+        final ItemStack stack = new ItemStack(state.getBlock(), 1);
         final Block block = stack.getItem() instanceof BlockItem ? ((BlockItem) stack.getItem()).getBlock() : null;
```

Comment:
This `block` initialization seems a bit roundabout now that `stack` seems to be guaranteed to be a `BlockItem`...